docs/Developers/Git_Basics.rst

```diff
@@ -163,6 +163,18 @@ Now that your message is written, you will have to save the file. Press escape
 to exit insert mode, and save the file (in Vim that is being done by pressing
 shift + Z twice).
 
+Run Coala
+------------------
+
+Now you can check if your commit messages and the code formattings are
+appropriate by using coala.
+If something goes wrong, coala will let you know. If this fails, CI
```

Comment:
I don't think we can assume everyone knows what CI stands for. Maybe change it to `continuous integration (CI)`?